### DISCUSSION TITLE
如何加入 [[roam/cn]] 中文社区？ #roamcult

### DISCUSSION THREAD

**Discussion by JimmyLv** [RoamCN/roamcult.vip#4] [#BestPractice 最佳实践]
Hi #RoamCN 伙伴们！Roam中文社区的小伙伴目前已经组织了B站、微信圈子、公共图谱和新的Discord平台，致力于更好建设RoamCN中文生态，和大家一起探索 @RoamResearch 的可能性，
<g-emoji class="g-emoji" alias="rocket" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f680.png">🚀</g-emoji>

* <g-emoji class="g-emoji" alias="bird" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f426.png">🐦</g-emoji> Twitter 地址：https://twitter.com/cn_roam
* <g-emoji class="g-emoji" alias="clapper" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f3ac.png">🎬</g-emoji> B 站地址：https://space.bilibili.com/599106362
* <g-emoji class="g-emoji" alias="spider_web" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f578.png">🕸</g-emoji> Roam 中文站公共图谱（Public Graph)：https://roamresearch.com/#/app/RoamCN/page/3TbMTyHMJ
* <g-emoji class="g-emoji" alias="hugs" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f917.png">🤗</g-emoji> Discord 永久地址：[https://discord.gg/stMehBs](https://t.co/Pes3bGfqEi?amp=1 "https://discord.gg/stMehBs")
* <g-emoji class="g-emoji" alias="o" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/2b55.png">⭕️</g-emoji> 微信圈子：直接微信「搜一搜」里面搜「Roam 中文社区」
* <g-emoji class="g-emoji" alias="ocean" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f30a.png">🌊</g-emoji> 知乎圈子：https://www.zhihu.com/club/1312820179727159296
* <g-emoji class="g-emoji" alias="postbox" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4ee.png">📮</g-emoji> 公众号：直接微信「搜一搜」里面搜「Roam Research 中文社区」
* <g-emoji class="g-emoji" alias="tv" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4fa.png">📺</g-emoji> 视频号：直接微信「视频号」里面搜「Roam Research」
* <g-emoji class="g-emoji" alias="headphones" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f3a7.png">🎧</g-emoji> 喜马拉雅：[http://xima.tv/7hMJvm?_sonic=0](https://t.co/RnJVvHHV2C?amp=1 "http://xima.tv/7hMJvm?_sonic=0")

**Comment by louisw00** [comment]
hello 各位roam research的中文使用的小伙伴们大家好 我是去年9月份开始使用roam的 我现在有个想法 但是我不会实现 想看下小伙伴们有没有谁有这个能力或者提出一些意见的 就是我能不能在当日的page下面自动生成我在其他页面建立的block 因为我们知道 在任何一个页面下建立一个block 都会有creation time这个creation time能不能自动 或者 有plugin能够将其总结到一个date的page里 这样每天我们即使产生很多笔记 但是也很方便我们总结和回顾 只是小弟的一点愚见 期待小伙伴们实现想法或指正想法
